src/com/owncloud/android/ui/activity/UploadFilesActivity.java

```diff
@@ -34,6 +34,7 @@
 import android.preference.PreferenceManager;
```

Comment:
Please update the comments in the bottom of the activity by including your name as @author and updating the year to 2018.